pkg/deploy/util/util.go

```diff
@@ -54,6 +54,16 @@ func LatestDeploymentNameForConfig(config *deployapi.DeploymentConfig) string {
 	return config.Name + "-" + strconv.Itoa(config.LatestVersion)
 }
 
+func ConfigSelector(config *deployapi.DeploymentConfig, list []api.ReplicationController) []api.ReplicationController {
```

Comment:
When #2692 lands this could be refactored to take a `name string` rather than a `*DeploymentConfig` to avoid having to create dummy objects as input.